src/control/tools/CircleHandler.cpp

```diff
@@ -45,7 +45,11 @@ void CircleHandler::drawShape(Point& c, const PositionInputData& pos) {
         {
             int signW = width > 0 ? 1 : -1;
             int signH = height > 0 ? 1 : -1;
-            width = std::max(width * signW, height * signH) * signW;
+            if (this->modControl) {
+                width = sqrt(pow(width, 2) + pow(height, 2));
+            } else {
+                width = std::max(width * signW, height * signH) * signW;
+            }
```

Comment:
This can be expressed in one line with:
```suggestion
            width = (this->modControl) ? sqrt(pow(width, 2) + pow(height, 2)) : std::max(width * signW, height * signH) * signW;
```

Reply:
Yes, I was aware of this. However that would take the line length to 128 characters, which makes it much more difficult to read. Having a look at the rest of the source, it seems that the convention is to stick to 120 characters or less. Perhaps a good middle ground is the following (although they will compile to the same thing anyway iirc):
```suggestion
            width = (this->modControl)
                    ? sqrt(pow(width, 2) + pow(height, 2))
                    : std::max(width * signW, height * signH) * signW;
```

Reply:
You can just clang-format your changes (which you must do anyway). The result is spread on two lines, separated by the double point (which is still on the first line).

Reply:
Ah, I'm sorry, I was not aware of this.